lib/Remotery.c

```diff
@@ -3450,6 +3450,8 @@ static void Base64_Encode(const rmtU8* in_bytes, rmtU32 length, rmtU8* out_bytes
 // domain. The author hereby disclaims copyright to this source code.
 //-----------------------------------------------------------------------------
 
+#if RMT_USE_INTERNAL_HASH_FUNCTION
```

Comment:
I've tried to copy the naming and usage style of the other options in the header file.

Not sure about the name. Is it clear enough?